### DISCUSSION TITLE
Plan a next release for WB

### DISCUSSION THREAD

**Discussion by lcaron** [eclipse/windowbuilder#154] [Ideas]
Hi

Since Java 16 WindowBuilder 19.7 is broken, and this bug has been fixed in the upcoming release, which is planned for September 2022.
I proposed to advance this date to mid-february or march.

What do you think ?

Take care,

Laurent

**Comment by wimjongman** [comment]
I will work on the release next week.

**Comment by lcaron** [comment]
Great ! Fill free to ask if you need help. We should also document the process (but I do not worry ;) )

**Comment by wimjongman** [comment]
Hi, I have started working on this. I will try to release it tomorrow.

**Comment by lcaron** [comment]
Great, thank you very much 👍

**Comment by wimjongman** [comment]
Done!
